tests/e2e/matmul/BUILD.bazel

```diff
@@ -84,9 +84,73 @@ X86_64_AVX512_BF16 = X86_64_AVX512 + [
     "+avx512bf16",
 ]
 
+# LLVMCPU, data-tiling.
+[iree_generated_trace_runner_test(
```

Comment:
Can this be folded into the existing dt_uk with a python list comprehension like we already do with the `for` loops on `lhs_rhs_type` and on `size` , adding a 3rd `for` loop there in the list comprehension?

Reply:
I think the answer is no because we need to skip `("bf16", "bf16")` and `("bf16", "f32"),` in the test. We disable them in nondt as well because of the same issue. I can try to fold it into `nondt`.

One question: do we want to set `target_cpu_features_variants` for `nondt` as same as `dt` and `dt+uk`? They are not set in `nondt` test suite at this moment.